LadybugTools_oM/ExecuteCommands/RunHeatPlotCommand.cs

```diff
@@ -0,0 +1,24 @@
+﻿using BH.oM.Adapter;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace BH.oM.LadybugTools
+{
+    [Description("Command that, when executed with the LadybugToolsAdapter, produces a heatmap from data in an epw file.\nOutput is a string of either the path to the image (if OutputLocation is not set) or the base 64 string representation of it.")]
+    public class RunHeatPlotCommand : IExecuteCommand
+    {
+        [Description("The path to an EPW file.")]
+        public virtual string EpwFile { get; set; } = "";
+
+        [Description("Key representing an hourly continuous collection in the EPW file to plot.")]
+        public virtual EpwKey EpwKey { get; set; } = EpwKey.Undefined;
```

Comment:
EPW is usually all capitals, should be consistent in property and object naming.